libraries/objectDefaultFiles/gl-worker.js

```diff
@@ -481,3 +485,43 @@ class ThreejsInterface {
         );
     }
 }
+
+// eslint-disable-next-line no-unused-vars
+class ThreejsFakeProxyInterface extends ThreejsInterface {
+    constructor(spatialInterface, injectThree) {
+        super(spatialInterface, injectThree);
+    }
+
+    main({width, height}) {
+        super.main({width, height});
+        this.canvas.width = width;
+        this.canvas.height = height;
+        document.body.appendChild(this.canvas);
+    }
+
+    render(now) {
+        this.preRender(now);
+
+        if (this.isProjectionMatrixSet) {
+            if (this.realRenderer && this.scene && this.camera) {
+                this.realRenderer.render(this.scene, this.camera);
+            }
+        }
+    }
+
+    setMatrixFromArray(matrix, array) {
```

Comment:
This shouldn't need to exist since it'll naturally call the superclass' implementation